### DISCUSSION TITLE
[Request] Any way to run from a batch folder - but NOT run the whole batch - just select ONE pose at random?

### DISCUSSION THREAD

**Discussion by Sacred-Reputation** [Mikubill/sd-webui-controlnet#1851] [General]
I have 30 poses I'm really happy with - lineart - in a folder. I can run the whole batch successfully, they all work great - I'm just looking for a way to run 1 at a time at random

**Comment by huchenlei** [comment]
The batch process is supposed to perform simple processes. For more customized need, we expect users to call A1111 API. See https://github.com/Mikubill/sd-webui-controlnet/wiki/API.
